src/router/storeSync/LayerParamConfig.class.js

```diff
@@ -244,6 +245,44 @@ function generateLayerUrlParamFromStoreValues(store) {
         .join(';')
 }
 
+// this one differs from the usual acceptedValues, as it handles each layer separately, telling the user
+// which layer won't render. It's basic, which means it will only tells the user when he gives a non
+// external layer that doesn't exist, or when he forgets the scheme for its external layer.
+function acceptedValues(store, query) {
+    const parsed = parseLayersParam(query)
+    const url_matcher = /https?:\/\//
+
+    parsed
+        .filter((layer) => !store.getters.getLayerConfigById(layer.id))
+        .forEach((layer) => {
+            if (
+                !layer.isExternal ||
+                (layer.baseUrl && !layer.baseUrl?.match(url_matcher)?.length > 0)
+            ) {
+                let value = ''
+                if (layer.isExternal) {
+                    value = `${layer.type}|`
+                }
+                value += layer.id
+                Object.entries(layer.customAttributes).forEach(
+                    ([attribute_name, attribute_value]) =>
+                        (value += `@${attribute_name}=${attribute_value}`)
+                )
+                if (!layer.visible || layer.opacity) {
+                    value += `,${layer.visible ? '' : 'f'},${layer.opacity ?? 1}`
+                }
```

Comment:
It doesn't make sense to rewrite the whole layer parameter here and it might not match the one from url due to default values and due to order of custom attributes, simply use the layer.id, keep it simple stupide.